mongodb/src/test/java/org/apache/calcite/adapter/mongodb/MongoDatabasePolicy.java

```diff
@@ -57,6 +59,14 @@ private MongoDatabasePolicy(MongoClient client, Closer closer) {
     closer.add(client::close);
   }
 
+  @Override public void beforeAll(ExtensionContext context) {
+
+  }
```

Comment:
Should we drop `BeforeAllCallback` then?

Reply:
well, I have dropped it.